### ISSUE TITLE
Complementary dataset usage in DenseStack MobRecon

### ISSUE THREAD

**Issue by jerryfzp** [SeanChenxy/HandMesh#27]
Hello Sean,

Thanks for releasing the DenseStack model! I got a quick question regarding the training recipe to achieve the 6.9 PJ shown in Table 5: to achieve such accuracy, do I need to use the complementary data in any stage of the training? I assume that since there is no marker associated with it, so it used the complementary data for pretraining and FreiHand only during fine-tuning. Could you please elaborate a bit on this? that would be super helpful.

Thanks!

**Comment by SeanChenxy** [comment]
Hi, to obtain 6.9 PJ on FreiHand, we use the complementary data for both pre-training and fine-tuning, as shown in the last row of Fig. 1.

**Comment by jerryfzp** [comment]
Cool! Thanks for the info 👍
